resources/nfdi4bioimage.yml

```
@@ -4483,4 +4483,19 @@ resources:
   url:
   - https://zenodo.org/records/13331351
   - https://doi.org/10.5281/zenodo.13331351
+  
+- authors: ''
+  license:
+  - CC-BY-SA-4.0
+  - CC-BY-NC-4.0
```

```suggestion
  - CC-BY-NC-SA-4.0
```
I quickly checked the footer of the website. It says this license